Create recommendations-ri-sp-coverage.md + the link to the how-to (#259)
hystax/optscale: ngui/ui/public/docs/recommendations-ri-sp-coverage.md

> <a href = 'https://hystax.com/how-to-use-optscale-to-optimize-ri-sp-usage-for-ml-ai-teams/'>How to use OptScale to optimize RI/SP usage for ML/AI teams </a>

Reviewer: ```suggestion
[How to use OptScale to optimize RI/SP usage for ML/AI teams](https://hystax.com/how-to-use-optscale-to-optimize-ri-sp-usage-for-ml-ai-teams/)
```
